RAGchain/benchmark/dataset/asqa.py

```diff
@@ -77,12 +77,16 @@ def ingest(self, retrievals: List[BaseRetrieval], db: BaseDB, ingest_size: Optio
         """
         ingest_data = deepcopy(self.content)
 
+        # Setting the evaluation size.
+        if self.eval_size is None:
+            eval_size = len(self.question)
+        else:
+            eval_size = self.eval_size
```

Comment:
`eval_size = self.eval_size if self.eval_size is not None else len(self.question)`